sdk/src/main/java/com/oursky/authgear/Authgear.kt

```diff
@@ -268,16 +269,22 @@ constructor(
 
     /**
      * Open the specific path on the authgear server.
+     *
+     * @param activity Activity for starting Chrome custom tab activity.
+     * @param page Page in Authgear Web UI.
      */
-    fun openUrl(path: String) {
-        core.openUrl(path)
+    fun openUrl(activity: Activity, path: String) {
```

Comment:
I wonder why we do not need to pass activity to authorize

Reply:
you mean the original implementation? This is because we can use `application` as context to start an activity, but this will require a new task (if not Android don't know where to put the activity). If we open an external browser than we can add `FLAG_ACTIVITY_NEW_TASK` and use `application.startActivity`

We can also do this for custom tab, but we will end up with two separate task (two item in recent app list) and I believe this is not wanted

Reply:
I mean both authorize and openURL should have the same behavior. They do not create a new, separate task.

Reply:
I see, authorize will create an intent to start `OAuthActivity`, and the custom tab uses `OAuthActivity` as context

Reply:
Oh, OAuthActivity is kind of magically. I am not sure if the authorization flow would still work if it is not only its own task stack.

Reply:
Any updates on this issue?

Reply:
You want to create a separate activity for open url just like OAuthActivity for authorize button?